src/PrestaShopBundle/Resources/views/Admin/Product/Include/form_combinations.html.twig

```diff
@@ -148,8 +148,9 @@
                 >
                 <label class="attribute-label" for="attribute-{{ attribute.id }}">
                   <span
-                    class="pretty-checkbox {% if attribute.color is empty %} not-color {% endif %}"
+                    class="pretty-checkbox {% if attribute.color is empty and attribute.texture is empty %} not-color {% endif %}"
                     {% if attribute.color is not empty %} style="background-color: {{ attribute.color }}" {% endif %}
+                    {% if attribute.texture is not empty %} style="content: url({{ attribute.texture }})" {% endif %}
```

Comment:
I think that it's possible to have color and texture in the same time, so we can't merge them into a single one. What do you think @maximebiloe ?